Offseason2022/src/main/java/frc/robot/commands/Climber5RotateIntoL3.java

```diff
@@ -13,17 +16,18 @@ public class Climber5RotateIntoL3 extends SequentialCommandGroup
 {
   public Climber5RotateIntoL3(Climber climber)
   {
+    setName("ClimberRotateIntoL3");
+
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+
+        // @formatter:off
+      new ParallelDeadlineGroup(
+          new WaitUntilCommand(climber::moveClimberDistanceIsFinished),
+          new ClimberMoveToHeight(climber, Height.NOCHANGE_HEIGHT), 
+      new ClimberSetGatehook(climber, false)

```

Comment:
Minor detail, but I think that ClimberSetGateHook should have the same indent level as the others, since it's included in that ParallelDeadlineGroup.